.idea/compiler.xml

```diff
@@ -39,17 +39,16 @@
       <profile name="Gradle Imported" enabled="true">
```

Comment:
It looks like this was included by accident. But maybe it's correct to update it if dependencies have changed (from an earlier pull request)?

Reply:
I only added it to force a rebuild in rec-layer, will remove